terra-jupyter-base/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.0.20 - 2021-06-09T16:18:53.734840Z
+
+- use deep learning images as base
```

Comment:
Maybe could specify which one (with a link) in the changelog, since users who extend images look at it